pkg/controllers/vrep/replication_reconciler.go

```diff
@@ -85,6 +88,12 @@ func (r *ReplicationReconciler) Reconcile(ctx context.Context, _ *ctrl.Request)
 		return ctrl.Result{}, nil
 	}
 
+	// no-op if Replicating is true (this is possible with async replication)
+	isReplicating := r.Vrep.IsStatusConditionTrue(v1beta1.Replicating)
+	if isReplicating {
```

Comment:
If this is only possible with async replication, do one more check - r.Vrep.Spec.Mode == replicationModeAsync - here.